numpy/lib/shape_base.py

```diff
@@ -992,27 +988,25 @@ def dsplit(ary, indices_or_sections):
     array([[[  0.,   1.,   2.,   3.],
             [  4.,   5.,   6.,   7.]],
            [[  8.,   9.,  10.,  11.],
-            [ 12.,  13.,  14.,  15.]]])
+            [12.,  13.,  14.,  15.]]])
```

Comment:
I think you need to touch the lines above too

Reply:
True, although refguide lets it through regardless. I'll do that now